readme.md

```diff
@@ -111,28 +113,47 @@ Create a language server for a unified ecosystem.
 
 Configuration for `unified-engine` and the language server.
 
+###### `options.processorName`
+
+The package ID of the expected processor (`string`, required, example:
+`'remark'`).
+Will be loaded from the local workspace.
+
+###### `options.processorSpecifier`
+
+The specifier to get the processor on the resolved module (`string`, optional,
+default: `'default'`).
+For example, remark uses the specifier `remark` to expose its processor and
+a default export can be requested by passing `'default'` (the default).
+
+###### `options.defaultProcessor`
+
+Optional fallback processor to use if `processorName` can’t be found
+locally in `node_modules` ([`Unified`][unified], optional).
+This can be used to ship a processor with your package, to be used if no
+processor is found locally.
+If this isn’t passed, a warning is shown if `processorName` can’t be found.
```

Comment:
Do we ever want this to be optional? It’s going to be enabled for `{remark,rehype,redot}-language-server`. Making it required later would be a breaking change, making it optional if someone asks for it would be semver minor.

Reply:
Is it going to be enabled? Reading https://github.com/neovim/nvim-lspconfig/pull/1606#discussion_r780224035 before adding this, I’m in favor of not adding the default by default

Reply:
Do you have opinions to add it?

Reply:
Not really. Doing so definitely has both pros and cons that IMO outweigh each other, so I’m fine with either option. I mainly thought you wanted to add it based on earlier conversations.

Reply:
I’m mostly 🤷‍♂️ as well, so let’s start without a default, and see if people want it!